gent_base.theme

```diff
@@ -654,6 +654,7 @@ function gent_base_preprocess_node(&$variables) {
 function gent_base_preprocess_links__node(&$variables) {
   if (isset($variables['links']['node-readmore'])) {
     $readmore = &$variables['links']['node-readmore'];
+    $readmore['link']['#options']['attributes']['rel'] = NULL;
```

Comment:
does this result in an empty `rel` attribute or is it deleted completely?

Reply:
It's deleted.